w3-filecoin.md

```diff
@@ -28,47 +28,47 @@ There are several roles in the authorization flow:
+| Dealer        | [Principal] identified by a DID, that arranges filecoin deals with storage providers. e.g. Spade |
+| Deal Tracker  | [Principal] identified by a DID, that tracks deals made by the Dealer |
 
 ### Storefront
 
-A _Storefront_ is a type of [principal] identified by a [`did:web`] identifier.
+A _Storefront_ is a type of [principal] identified by a DID (typically a [`did:web`] identifier).
 
 A Storefront facilitates data storage services to applications and users, getting the requested data stored into Filecoin deals asynchronously.
 
 ### Aggregator
 
-An _Aggregator_ is a type of [principal] identified by a `did:key` identifier.
+An _Aggregator_ is a type of [principal] identified by a DID (typically a `did:key` identifier).
 
 An Aggregator facilitates data storage into Filecoin deals by aggregating smaller data (Filecoin Pieces) into a larger piece that can effectively be stored with a Filecoin Storage Provider using [Verifiable Data Aggregation
 ](https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0058.md).
 
 ### Dealer
 
-A _Dealer_ is a type of [principal] identified by a `did:key` identifier that arranges deals for the aggregates submitted by _Storefront_.
+A _Dealer_ is a type of [principal] identified by a DID (typically a `did:key` identifier) that arranges deals for the aggregates submitted by _Storefront_.
 
-### Chain Tracker
+### Deal Tracker
 
-A _Chain Tracker_ is a type of [principal] identified by a `did:key` that tracks the filecoin chain to keep a view of successful deals.
+A _Deal Tracker_ is a type of [principal] identified by a DID (typically a `did:key` identifier) that follows the filecoin chain to keep track of successful deals.
 
 # Protocol
 
 ## Overview
 
-A Storefront is web2 to web3 bridge. It ingests user/application data through a conventional web2 interface and replicates it across various storage systems, including Filecoin Storage Providers. Content supplied to a Storefront can be of arbitrary size, while (Filecoin) Storage Providers demand large (>= 16GiB) content pieces.  To align supply and demand requirements, the aggregator accumulates supplied content pieces into a larger verifiable aggregate pieces per [FRC-0058](https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0058.md) that can be stored by Storage Providers.
+A Storefront is a service that ensures content addressed user/application data is stored perpetually on the decentralized web. A Storefront ingests user/application data and replicates it across various storage systems, including Filecoin Storage Providers. Content supplied to a Storefront can be of arbitrary size, while (Filecoin) Storage Providers demand large (>= 16GiB) content pieces.  To align supply and demand requirements, the aggregator accumulates supplied content pieces into a larger verifiable aggregate pieces per [FRC-0058](https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0058.md) that can be stored by Storage Providers.
```

Comment:
Do we need to say "web2 to web3 bridge"?

Reply:
I think proposed version reads better.